tests/dialects/test_starrocks.py

```diff
@@ -160,3 +160,65 @@ def test_analyze(self):
         self.validate_identity(
             "ANALYZE TABLE TBL UPDATE HISTOGRAM ON c1, c2 WITH ASYNC MODE WITH 5 BUCKETS PROPERTIES ('prop1'=val1)"
         )
+
+    def test_is_bool_allowed(self):
```

Comment:
We don't need these tests, a single one suffices. I'll get the PR in and clean it up.